SofaKernel/extlibs/newmat/CMakeLists.txt

```diff
@@ -41,7 +41,12 @@ set(SOURCE_FILES
 add_library(${PROJECT_NAME} STATIC ${HEADER_FILES} ${SOURCE_FILES})
 target_include_directories(${PROJECT_NAME} PUBLIC "$<BUILD_INTERFACE:${CMAKE_CURRENT_SOURCE_DIR}>")
 target_include_directories(${PROJECT_NAME} PUBLIC "$<INSTALL_INTERFACE:include>")
-target_compile_definitions(${PROJECT_NAME} PRIVATE "use_namespace")
+target_compile_definitions(${PROJECT_NAME} PUBLIC "use_namespace")
+if(NOT DEFINED SOFA_WITH_DOUBLE)
```

Comment:
Some points to keep in mind regarding newmat:
* It has not been updated for 9 years or so now.
* It does not offer CMakeLists officially.
* Configuration is meant to be done inside newmat/include.h
Thus the CMakeLists available has been created by Sofa developers, and newmat internals have been modified by Sofa developers too. So if there is an update, the include.h will have to be updated, and the CMakeLists file will remain the same.

Therefore is it still required to have it above in the build tree ? (It would then be located in .\sofa\src\SofaKernel\extlibs\CMakeLists.txt )